### ISSUE TITLE
Lava cooling to basalt

### ISSUE THREAD

**Issue by anoomolu** [pcal43/trailblazer#16]
Hiya, love the idea of this mod and have been using High-Speed Rail for a while now.

Had a quick feature-request (I guess it's more like an adjustment to default behaviour) - lava cools to basalt irl, perhaps the lavawalking feature could be adjusted to lava -> basalt by default instead?

**Comment by pcal43** [comment]
yeah that's a good call.  i'll update the default config to do that.
